ci/jenkins/test-vmc.sh

```diff
@@ -294,7 +316,9 @@ function run_e2e {
     echo "=== Master node ip: ${master_ip} ==="
     sed -i "s/MASTERNODEIP/${master_ip}/g" $GIT_CHECKOUT_DIR/test/e2e/infra/vagrant/ssh-config
     echo "=== Move kubeconfig to master ==="
-    ssh -q -o StrictHostKeyChecking=no -i $GIT_CHECKOUT_DIR/jenkins/key/antrea-ci-key -n capv@${master_ip} "mkdir -p .kube"
+    ssh -q -o StrictHostKeyChecking=no -i $GIT_CHECKOUT_DIR/jenkins/key/antrea-ci-key -n capv@${master_ip} "if [ ! -e ".kube" ]; then
```

Comment:
Do we need to add a `\` before `"`?

Reply:
`mkdir -p .kube` won't work for OS other than ubuntu?

Reply:
The actual command running is: `ssh .....  'if [ ! -e .kube ]; then ... fi'` according to the logs, so we don't need to add `\`.

Reply:
It's not that `mkdir -p .kube` won't work. It's because e2e, network policy and conformance tests will create this directory more than once.

Reply:
I think it is the same. As for me, `mkdir -p .kube` ensures that the directory exists and it is only one line without performance issue.

Reply:
Create the directory with `mkdir -p .kube` when that directory already exists would lead to error like this: `mkdir: cannot create directory – File exists`.

Reply:
Does it come from OS other than ubuntu? It should work on ubuntu. Could you verify if there are any options like `-p` to ignore this error? If not, I am ok with your implementation.

Reply:
Yes, `mkdir -p` works on ubuntu/darwin, but fails on centos and photon. 

Checked on the CentOS online man page: http://man.linuxtool.net/centos7/u0/man/1_mkdir.html, it says: 
`−p, −−parents.   no error if existing, make parent directories as needed`, while `cannot create directory file exists` seems a common problem on some Linux OSs: https://stackoverflow.com/questions/14496897/mkdir-p-fails-when-directory-exists. Considering all the references, I think it's better to remain this implementation.

Reply:
it's weird, it should only fail if there is already a *file* by that name... And then I would expect the subsequent `scp` command to fail

Reply:
Please note that a directory is also a kind of file on Linux.

Reply:
Agree with Antonin. I'd propose we choose from one of the following style:

- `mkdir -p`
-  `if [ ! -d ".kube" ]; then mkdir .kube; fi`  (change the `-e` in the current code to `-d`)